lib/backend/lite/lite.go

```diff
@@ -337,16 +330,30 @@ func (l *Backend) createSchema() error {
 		}
 	}
 
-	return nil
-}
+	rows, err := l.db.QueryContext(l.ctx, `PRAGMA table_info('kv');`)
+	if err != nil {
+		return trace.Wrap(err)
+	}
 
-func (l *Backend) newLease(item backend.Item) *backend.Lease {
-	var lease backend.Lease
-	if item.Expires.IsZero() {
-		return &lease
+	columns := make(map[string]struct{})
+	for rows.Next() {
+		var tmp, name, null sql.NullString
+		err := rows.Scan(&tmp, &name, &tmp, &null, &tmp, &tmp)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+		columns[name.String] = struct{}{}
+	}
+
+	if _, ok := columns["revision"]; !ok {
+		const addRevision = `ALTER TABLE kv ADD COLUMN revision TEXT NOT NULL DEFAULT "";`
+		if _, err := l.db.ExecContext(l.ctx, addRevision); err != nil {
+			l.Errorf("Failing schema step: %v, %v.", addRevision, err)
+			return trace.Wrap(err)
+		}
 	}
-	lease.Key = item.Key
-	return &lease
+
+	return nil
```

Comment:
If we're not ok with just attempting to add the column and ignoring the error if the column exists, why not do `select exists(select 1 from pragma_table_info('kv') where name = 'revision');` rather than parsing the entire table info?